gpkit/nomials.py

```diff
@@ -492,22 +490,14 @@ def __pow__(self, other):
 
 
 class Constraint(Posynomial):
-    """A constraint of the general form monomial > posynomial
-    Stored internally (exps, cs) as a single Posynomial (1 >= self)
+    """A constraint of the general form posynomial <= monomial
```

Comment:
I decided to follow the usual convention of posynomial <= 1 given in most descriptions of GP. But willing to switch it to the other direction if you feel strongly -- just say so below and I'll make the switch.